crates/routes/Cargo.toml

```diff
@@ -32,5 +32,16 @@ serde = { workspace = true }
 url = { workspace = true }
 tracing = { workspace = true }
 tokio = { workspace = true }
+futures-util.workspace = true
 http.workspace = true
+diesel.workspace = true
```

Comment:
Would probably be good to use the same style for each.

Reply:
Yes but I dont want to change everything manually, and taplo doesnt have such an option.

https://taplo.tamasfe.dev/configuration/formatter-options.html